README.md

```diff
@@ -1 +1,167 @@
-# Ruby-Capstone
\ No newline at end of file
+# Ruby-Capstone
+
+<a name="readme-top"></a>
+<div align="center">
+    <img src="./code.png" alt="logo" width="700"  height="auto" />
+  <br/>
+  <h3><b>Ruby Capstone</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 TDD Project](#tdd-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [📝 License ](#-license-)
+
+# 📖 TDD Project <a name="about-project">TDD Project</a>
```

Comment:
Kindly give a proper title to your project. I believe this is the title of your last project. Remember, it is super important to have a professional and clean README file.

Reply:
Hello @muneebulrehman 

Thank you so much for your kind feedback. We will make the required changes and shall again ask for a review.

Regards